### PR TITLE
Feature/LMSNEXT-530 ENV file Step for New Services 

### PR DESCRIPTION
**Pull request by hperera-acentura**
- Introduced the `Create .env File if Exists`  step to service pipelines to support new service deployments and their configuration management approach. 
- Pipelines were tested and verified 

# Note
**With the implementation of this modification, there will be no impact on existing legacy service deployments. This is because the newly introduced step is designed to be executed only under the specific condition that the .env.dev, .env.stg, and .env.prod environment files are present. If these files are not detected, the step will be bypassed, ensuring that the legacy service deployments remain unaffected.**

### CODE REVIEW

**Review by SasankaAc — APPROVED** [review]
LGTM

File: .github/workflows/services_build_deploy_dev.yml [hunk 1]
```diff
@@ -99,6 +102,14 @@ jobs:
           GH_TOKEN: ${{ secrets.SHIELDS_GH_TOKEN }}
           REPO_NAME: ${{ github.repository }}
 
+      - name: Create .env File if Exists
+        run: |
+          if [ -f ".env.$APP_ENV" ]; then
+            mv -f ".env.$APP_ENV" .env
+            rm -f .env.*
+            cat .env
```
Comment (Lazzolla) [inline .github/workflows/services_build_deploy_dev.yml:110]: Should we expect .env files shared on github repositories?

**Review by Lazzolla — COMMENTED** [review]
I'm not sure I understand the purpose of these changes.
I reviewed the implementation of the secret configuration at:
https://github.com/Cell-Ed/cei-svc-chatbot/blob/develop/src/config/secret-configuration.ts#L45
and
https://github.com/Cell-Ed/cei-svc-chatbot/blob/develop/src/app.module.ts#L3
It looks like the secrets will be stored in memory.

This PR doesn't have a Jira ticket associated, but I see LMSNEXT-530 in the title. Unfortunately, at the time of this review, that ticket is empty: https://celled.atlassian.net/browse/LMSNEXT-530
@hperera-acentura, could you provide more details about the changes in this PR and their purpose?

File: .github/workflows/services_build_deploy_dev.yml (hunk 1 above)
Comment (Lazzolla) [reply .github/workflows/services_build_deploy_dev.yml:110]: I see now the connection of this changes with this PR: https://github.com/Cell-Ed/cei-svc-chatbot/pull/10.
I don't think this is the best pattern to shared environment variables in our secrets. 
I suggest to leverage the secrets pattern that it's already in use. I'll provide more details on https://github.com/Cell-Ed/cei-svc-chatbot/pull/10

Comment (hperera-acentura) [reply .github/workflows/services_build_deploy_dev.yml:110]: Sorry for the inconvenience occurred with lack of description the [LMSNEXT-530](https://celled.atlassian.net/browse/LMSNEXT-530). 

Expecting to do all the required changes to deploy chatbot API into the staging environment under same jira.

Comment (hperera-acentura) [reply .github/workflows/services_build_deploy_dev.yml:110]: Review comment is incorporated with the discussion we had. Thank you very much for your inputs and suggested [hybrid approach](https://celled.atlassian.net/wiki/x/AwDOMw) found with the hyperlink

**Pull request merged**